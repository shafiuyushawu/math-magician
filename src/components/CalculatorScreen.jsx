function CalculatorScreen() {
    return <div className=" rounded-sm h-10 w-[20%] bg-slate-400 text-right text-white p-2">0</div>
}
export default CalculatorScreen