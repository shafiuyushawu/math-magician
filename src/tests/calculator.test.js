import { render } from '@testing-library/react';
import CalculatorScreen from '../components/CalculatorScreen';

describe('CalculatorScreen', () => {
  it('renders correctly', () => {
    const objState = {
      total: '10',
      next: '5',
      operation: '+',
    };
    const { container } = render(<CalculatorScreen objState={objState} />);
    expect(container).toMatchSnapshot();
  });
  it('test buttons', () => {

  });
});
