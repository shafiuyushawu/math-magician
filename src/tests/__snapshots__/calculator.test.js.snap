// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorScreen renders correctly 1`] = `
<div>
  <div
    class="rounded-sm h-20 w-[80%] bg-slate-400 text-5xl text-right text-white p-2"
    data-testid="calculator-screen"
  >
    0
  </div>
</div>
`;
